src/humancellatlas/data/metadata/api.py

```diff
@@ -459,7 +459,27 @@ def __init__(self, json: JSON):
 
 @dataclass(init=False)
 class AnalysisProtocol(Protocol):
-    pass
+    describedBy: str
+    computational_method: str
+    provenance_document_id: str         # Entity self.document_id ?
+    provenance_submission_date: str
+    provenance_update_date: Optional[str]
+    protocol_core_protocol_id: str      # Protocol self.protocol_id ?
+    protocol_type_text: str
+
```

Comment:
Two of the AnalysisProtocol fields are already fetched by parent classes Protocol and Entity. Should these fields be removed from the AnalysisProtocol class to avoid duplication, or kept in AnalysisProtocol to make sure these fields exist for this class?